packages/upgrade/transforms/imports-to-unified-package.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
```

Comment:
For the transform name, would it be possible to use a name like: `update-carbon-react-import-to-scoped` or something more specific?

Reply:
@joshblack Yes! I like that one. I thought mine wasn't that great but ran with it anyway. Thanks for the suggestion 👍

Reply:
I might go for `update-carbon-components-react-import-to-scoped` actually to pinpoint `carbon-components-react`

Reply:
Updated